docs/troubleshooting-guide.md

```diff
@@ -22,55 +22,11 @@ Note that some of the screenshots might be outdated, but the instructions will r
   **SOLUTION**: You can re-run the install command again without any side effect. Make sure to choose to update the `PATH` variable this time. Alternatively, you can use other appropriate methods to update your `PATH` variable to include the `/bin` sub-folder of the SDK folder.<br>
```

Comment:
Troubleshooting issues are often edge cases or common cases, collapsing them into the IDE section is unnecessarily causing bloat to the IDE document.